src/k8s-engine/minikube.js

```diff
@@ -112,6 +112,15 @@ class Minikube extends EventEmitter {
       // TODO: Handle the difference between changing version where a wipe is needed
       // and upgrading. All if there was a change.
       args.push('--kubernetes-version=' + this.cfg.version);
+      const memoryInGB = this.cfg.memoryInGB;
+      if (memoryInGB !== 2) {
```

Comment:
I _still_ don't understand this: why do we _not_ provide a value if it's 2? Don't we still need it even if a cluster already exists (it would just not do anything)?

Reply:
Oh yeah, that's wrong. What if you're currently using a different value for CPUs or memory?